tools/check_jsonnet

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+success=1
+JSONNETS=$(git ls-files schedule/ test_data/ | grep '.json\?net$' | xargs echo)
```

Comment:
the folder to search for is only data/
the extension is only .jsonnet not .json

Reply:
updated